README.md

```diff
@@ -421,7 +421,8 @@ You can also fork the project on github and run your fork's [build workflow](.gi
                                      (e.g. 50k or 44.6m)
     --date DATE                      Download only videos uploaded on this date.
                                      The date can be "YYYYMMDD" or in the format
-                                     "(now|today)[+-][0-9](day|week|month|year)(s)?"
+                                     "(now|today|yesterday|date)[+-][0-9]+
+                                      (microsecond|second|minute|hour|day|week|month|year)(s)?"
```

Comment:
1. This is auto-generated. The text should be edited in options.py
2. The other syntax while technically working, is meant only for internal use and not to be user-facing. Especially the hour, min etc won't work as you expect since the option only looks at the date and not timestamp

Reply:
1. Reverted and updated in options.py.
2. The reason I looked into it was because I had to validate all possible options. If it's technically correct, I think it should be there.

Reply:
I have instead added validation to the reasonable values